iam-login-service/src/main/java/it/infn/mw/iam/notification/TransientNotificationFactory.java

```diff
@@ -245,8 +247,22 @@ public IamEmailNotification createGroupMembershipRejectedMessage(IamGroupRequest
   protected IamEmailNotification createMessage(String template, Map<String, Object> model,
       IamNotificationType messageType, String subject, List<String> receiverAddress) {
 
+    VelocityEngine ve;
+
+    if(properties.getUseCustomTemplates()) {
```

Comment:
Please use these formatting conventions for Java code:
https://github.com/italiangrid/codestyle/blob/master/eclipse-google-java-codestyle-formatter.xml

the formatting rules are for Eclipse, but you should easily get a converter if you use idea.